### DISCUSSION TITLE
[@types/d3-array] Signatures for d3.group/d3.groups with limited amount of "key" parameters

### DISCUSSION THREAD

**Discussion by rodrigodivino** [DefinitelyTyped/DefinitelyTyped#57101] [Issues with a @types package]
The type definitions for the methods `d3.group` and `d3.groups` only support up to three parameters, but the implementation supports multiple parameters with a spread operator. Is this caused by some limitation, or is this something that can be adjusted?

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment]
I'm not aware of a limitation, they were likely just written when TypeScript was more limited in this regard. If you could open a PR that would be great. Thanks!
